src/test/scala/chiselTests/experimental/DataView.scala

```diff
@@ -546,6 +546,40 @@ class DataViewSpec extends ChiselFlatSpec {
     chirrtl should include("dataOut <= vec[addrReg]")
   }
 
+  it should "allow views between reset types" in {
+    class A extends Bundle {
+      val uint = AsyncReset()
```

Comment:
```suggestion
      val uint = Bool()
```
I think you meant to make this a Bool